CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -299,6 +299,17 @@ function PlayerHospital:onEndDay()
     self:dailyAdviceChecks()
   end
 
+  -- Occasionally unstaffed rooms with a queue will call for staff
+  for _, room in pairs(self.world.rooms) do
+    if math.random(1, 20) == 1 and #room.door.queue > 0 and not room.staff_member and
+        not room:testStaffCriteria(room:getRequiredStaffCriteria()) then
+          self.world.dispatcher:callForStaff(room)
+          break
```

Comment:
I understand what you're trying to do here, but this looks like there are 2 nested constructs, while in reality there is only 1 construct.
To solve it, do
``` suggestion
    if math.random(1, 20) == 1 and #room.door.queue > 0 and not room.staff_member and
        not room:testStaffCriteria(room:getRequiredStaffCriteria()) then
      self.world.dispatcher:callForStaff(room)
      break
```
The 'then' part is indented as normal, while the 'not' line has 'weird' indenting, sending a signal 'watch out!'